img: Add dark-mode logo (Fixes #4)

ochosi opened the pull request:
This commit sets a dark-mode logo.
While our logo is an svg, the usual approach of resorting to a ThemedImage doesn't work.
Instead, the navbar's srcDark field needs to be set to a different logl.

See

- https://docusaurus.io/docs/next/markdown-features/assets#themed-images
- https://docusaurus.io/docs/api/themes/configuration#navbar-logo

Note: I'm aware that #4 refers to more than just the logo, but I feel the logo is the main pain point. All the other svgs are still legible.

larskarlitski commented on feed.fedoraplanet.xml:
```diff
@@ -0,0 +1,19 @@
+  <published>{{ entry.date | date_to_xmlschema }}</published>
+  <updated>{{ entry.last_modified_at | default: entry.date | date_to_xmlschema }}</updated>
+  <author><name>{{ site.author | xml_escape }}</name></author>
+  <link href="{{ entry.url | absolute_url }}" rel="alternate" type="text/html" title="{{ entry.name | xml_escape }}" />
```
The `title` attribute ends up being empty, because `entry.name` does not exist. Is your idea to use `entry.caption` here, as defined in the front matter of the news entries in `_collections/_news`?

dvdhrm replied:
Yes! Thanks for catching that.

larskarlitski commented on feed.xml:
```diff
@@ -0,0 +1,17 @@
+  <published>{{ entry.date | date_to_xmlschema }}</published>
+  <updated>{{ entry.last_modified_at | default: entry.date | date_to_xmlschema }}</updated>
+  <author><name>{{ site.author | xml_escape }}</name></author>
+  <link href="{{ entry.url | absolute_url }}" rel="alternate" type="text/html" title="{{ entry.name | xml_escape }}" />
```
Same as above here.

dvdhrm replied:
(Also fixed.)

dvdhrm commented:
> The [W3C feed validator](https://validator.w3.org/feed/) complains that two entries contain the same `updated` field, but only as a recommendation.

Right. Not sure what to do there. We do not have detailed "updated" times, so they will often end up being the same. I will just ignore it, unless someone has suggestions what we could do there?

I fixed the `name -> caption` bug and force-pushed! Thanks a lot for the review!

ochosi committed:
img: Add dark-mode logo (Fixes #4)

This commit sets a dark-mode logo.
While our logo is an svg, the usual approach of resorting to a
ThemedImage doesn't work.
Instead, the navbar's srcDark field needs to be set to a different logl.

See
https://docusaurus.io/docs/next/markdown-features/assets#themed-images
https://docusaurus.io/docs/api/themes/configuration#navbar-logo
